packages/core/__tests__/diff/schema.test.ts

```diff
@@ -803,3 +801,161 @@ test('adding root type should not be breaking', async () => {
     ]
   `);
 });
+
+test('null old schema', async () => {
+  const schemaA = null;
+
+  const schemaB = buildSchema(/* GraphQL */ `
+    type Query {
+      foo: String
+    }
+
+    type Subscription {
+      onFoo: String
+    }
+  `);
+
+  const changes = await diff(schemaA, schemaB);
+  expect(changes).toMatchInlineSnapshot(`
+    [
+      {
+        "criticality": {
+          "level": "NON_BREAKING",
+        },
+        "message": "Schema query root has changed from 'unknown' to 'Query'",
+        "meta": {
+          "newQueryTypeName": "Query",
+          "oldQueryTypeName": "unknown",
```

Comment:
Isn't unknown also a legit name that the root Query type could be named after? Maybe it should be `null` and not a string instead.

```
type unknown {
  foo: Int
}

schema {
  query: unknown
}
```